helix-term/src/ui/mod.rs

```diff
@@ -91,13 +91,14 @@ pub fn regex_prompt(
     )
 }
 
-pub fn file_picker(root: PathBuf) -> FilePicker<PathBuf> {
+pub fn file_picker(root: PathBuf, config: helix_view::editor::Config) -> FilePicker<PathBuf> {
```

Comment:
You should pass a reference instead.

Reply:
Good note, will change in new commit.

Thanks!